lib/id3c/cli/command/manifest.py

```diff
@@ -222,13 +257,23 @@ def parse_using_config(config_file):
             LOG.error(f"Required key «{key}» missing from config {config}")
             raise key from None
 
+        if config.get("sample_column") is None and config.get("collection_column") is None:
+            raise KeyError("The configuration must indicate the sample column, the collection \
+                column, or both.")
```

Comment:
I wonder if there's an edge case where a user may accidentally specify an empty string for sample_column and collection_column, in which case we would not raise an error

Reply:
Since the YAML reader returns a string I was able to use `not`. Good call out. That was a simple enough change to cover both no entry and an empty string entry.